ocaml/idl/datamodel_repository.ml

```diff
@@ -94,6 +95,28 @@ let introduce_bundle =
     ~allowed_roles:(_R_POOL_OP ++ _R_CLIENT_CERT)
     ()
 
+let introduce_remote_pool =
+  call ~name:"introduce_remote_pool" ~in_oss_since:None ~lifecycle:[]
+    ~doc:"Add the configuration for a new remote pool repository"
+    ~params:
+      [
+        (String, "name_label", "The name of the repository")
+      ; (String, "name_description", "The description of the repository")
+      ; ( String
+        , "address"
+        , "The host name or a string representation of the IP address of the \
+           coordinator of the remote pool repository"
+        )
```

Comment:
> ... of the coordinator of the remote pool repository

I think it should be `the coordinator of the remote pool`. Same comments for other similar phrases.